test/l4lb/test.sh

```diff
@@ -4,6 +4,7 @@ set -eux
 
 IMG_OWNER=${1:-cilium}
 IMG_TAG=${2:-latest}
+PULL_REQUEST_CHECKOUT=${3:-latest}
```

Comment:
I think we should default to `../..` for the directory here for backwards compatibility:

```suggestion
PULL_REQUEST_CHECKOUT=${3:-../..}
```